source/numerics/vector_tools_rhs.inst.in

```diff
@@ -14,25 +14,24 @@
 // ---------------------------------------------------------------------
 
 
-for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension :  SPACE_DIMENSIONS)
+for (VEC : SERIAL_VECTORS; deal_II_dimension : DIMENSIONS; deal_II_space_dimension :  SPACE_DIMENSIONS)
```

Comment:
unless you test that it compiles with complex-valued PETSc, probably it's better to instantiate with `REAL_SERIAL_VECTORS` instead.